### PR TITLE
Cleaned up TTTT

### PR DESCRIPTION
**Pull request by SansPapyrus683**
No templates used (nor `bits/stdc++`), but I hope that's fine.

### CODE REVIEW

File: solutions/USACO open18-team-tic-tac-toe.mdx [hunk 1]
```diff
@@ -15,229 +15,48 @@ author: Jesse Choe, Benjamin Qi
...
-#define ins insert
-#define ft front()
-#define bk back()
-#define pb push_back
-#define eb emplace_back
-#define pf push_front
+#include <fstream>
+#include <vector>
+#include <string>
+#include <set>
```
Comment (HarryWangATX) [inline solutions/USACO open18-team-tic-tac-toe.mdx:21]: Just use stdhc lmao no include clutter

File: solutions/USACO open18-team-tic-tac-toe.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 id: usaco-831
 source: USACO Bronze 2018 Open
 title: Team Tic Tac Toe
-author: Jesse Choe, Benjamin Qi
+author: Jesse Choe, Kevin Sheng, Benjamin Qi
```
Comment (HarryWangATX) [inline solutions/USACO open18-team-tic-tac-toe.mdx:5]: 🤦

Comment (SansPapyrus683) [reply solutions/USACO open18-team-tic-tac-toe.mdx:5]: so should i? i mean i did so something lol

**Review by thecodingwizard — CHANGES_REQUESTED** [review]
Use tabs instead of 4 spaces :)

Also, I think we should standardize using `using namespace std;`...

**Comment by SansPapyrus683** [comment]
Oh yeah, forgot about the tabs part-
As for the `namespace std`, I don't feel like standardizing would be such a big deal anyways.

File: solutions/USACO open18-team-tic-tac-toe.mdx [hunk 3]
```diff
@@ -15,229 +15,45 @@ author: Jesse Choe, Benjamin Qi
...
 int main() {
-	setIO("tttt");
-	re(board);
-	F0R(i,3) ins({{i,0},{i,1},{i,2}}); // insert rows
-	F0R(i,3) ins({{0,i},{1,i},{2,i}}); // insert cols
-	ins({{0,0},{1,1},{2,2}}); // insert diagonals
-	ins({{2,0},{1,1},{0,2}});
-	ps(sz(winners[1]));
-	ps(sz(winners[2]));
+	std::ifstream read("tttt.in");
```
Comment (maggie-j-liu) [inline solutions/USACO open18-team-tic-tac-toe.mdx:38]:
```suggestion
	ifstream read("tttt.in");
```

File: solutions/USACO open18-team-tic-tac-toe.mdx [hunk 4]
```diff
@@ -15,229 +15,45 @@ author: Jesse Choe, Benjamin Qi
...
+	}
+	// insert columns
+	for (int i = 0; i < WIDTH; i++) {
+		insert({{0, i}, {1, i}, {2, i}});
+	}
+	// insert the 2 diagonals
+	insert({{0, 0}, {1, 1}, {2, 2}});
+	insert({{2, 0}, {1, 1}, {0, 2}});
+
+	std::ofstream written("tttt.out");
```
Comment (maggie-j-liu) [inline solutions/USACO open18-team-tic-tac-toe.mdx:54]:
```suggestion
	ofstream written("tttt.out");
```

**Review by maggie-j-liu — COMMENTED** [review]
You are `using namespace std;`...

**Comment by SansPapyrus683** [comment]
Yeah, forgot about that. At the time I thought it would allow it to be more explicit.

File: solutions/USACO open18-team-tic-tac-toe.mdx (hunk 4 above)
Comment (thecodingwizard) [reply solutions/USACO open18-team-tic-tac-toe.mdx:54]: not resolved?

**Review by darren-yao — APPROVED** [review]
im just gonna assume that content authors submitted the code to make sure it works beforehand

**Pull request merged**